src/main/java/de/rub/nds/scanner/core/passive/ExtractedValueContainer.java

```diff
@@ -17,18 +17,30 @@
 
 public class ExtractedValueContainer<ValueT> {
 
-    private final List<ValueT> extractedValueList;
+    @JsonTypeInfo(
+            use = JsonTypeInfo.Id.CLASS,
+            include = JsonTypeInfo.As.PROPERTY,
+            property = "@class")
+    private List<Object> extractedValueList;
 
-    private final TrackableValue type;
+    private TrackableValue type;
```

Comment:
Ideally we have this final again - I dont know if we have a unit test for deserialization in this project - if not we need one.